src/CodeStyle/Core/CodeFixes/FormattingCodeFixProvider.cs

```diff
@@ -77,14 +77,14 @@ public FixAll(AbstractFormattingCodeFixProvider formattingCodeFixProvider)
                 _formattingCodeFixProvider = formattingCodeFixProvider;
             }
 
-            protected override string CodeActionTitle => CodeStyleResources.Fix_formatting;
+            protected override string GetCodeActionTitle(FixAllContext fixAllContext) => CodeStyleResources.Fix_formatting;
 
-            protected override async Task<SyntaxNode> FixAllInDocumentAsync(FixAllContext fixAllContext, Document document, ImmutableArray<Diagnostic> diagnostics)
+            protected override async Task<Document> FixAllInDocumentAsync(FixAllContext fixAllContext, Document document, ImmutableArray<Diagnostic> diagnostics)
             {
                 var options = await _formattingCodeFixProvider.GetOptionsAsync(document, fixAllContext.CancellationToken).ConfigureAwait(false);
                 var syntaxRoot = await document.GetSyntaxRootAsync(fixAllContext.CancellationToken).ConfigureAwait(false);
                 var updatedSyntaxRoot = Formatter.Format(syntaxRoot, _formattingCodeFixProvider.SyntaxFormattingService, options, fixAllContext.CancellationToken);
-                return updatedSyntaxRoot;
+                return document.WithSyntaxRoot(updatedSyntaxRoot);
```

Comment:
1. why this change?
2. can we make this change independent of this PR?

Reply:
Yep, reverted now.